package.json

```diff
@@ -18,11 +18,16 @@
     "lint": "eslint 'src/**/*.js'",
     "prettier": "prettier --write '{src/**/,scripts/**/,}*.js'",
     "prettier:check": "prettier-check '{src/**/,scripts/**/,}*.js'",
-    "test": "jest --no-cache --coverage",
+    "test": "jest --no-cache --coverage src",
     "test:ci": "run-p test prettier:check lint",
+    "test:watch": "jest --watch --runInBand src",
```

Comment:
Would like to see `runInBand` removed from here - if you're not debugging, this is just going to slow down test runs.